src/test/java/kitchenpos/MenuTest.java

```diff
@@ -0,0 +1,357 @@
+        purgomalumClient = new FakeDefaultPurgomalumClient();
+        menuService = new MenuService(menuRepository, menuGroupRepository, productRepository, purgomalumClient);
+    }
+
+    @DisplayName(value = "메뉴 등록 기능")
+    @Nested
+    class MenuCreateTest {
+        private static final BigDecimal 후라이드치킨_MINUS_PRICE = new BigDecimal(-10);
+        private static final String 후라이드치킨_PROFANITY_MENU_NAME = "fucking 치킨메뉴";
+        private static final int MINUS_QUANTITY = -1;
+
+
+
+        @DisplayName(value = "메뉴를 등록할 수 있다.")
+        @Test
+        void createMenuTest() {
+            Mockito.clearInvocations(menuRepository, productRepository);
+            Product product = createProduct();
+            productRepository.save(product);
+            MenuGroup menuGroup = createMenuGroup();
+            menuGroupRepository.save(menuGroup);
+            MenuProduct menuProduct = createMenuProduct(product, 1, 후라이드치킨_PRODUCT_UUID);
+            Menu menuRequest = createMenu();
+            Menu menu = menuService.create(menuRequest);
+            //행위검증
+            verify(menuRepository, times(1)).save(ArgumentCaptor.forClass(Menu.class).capture());
+            verify(productRepository, times(1)).findAllByIdIn(Mockito.anyList());
+            verify(productRepository, times(1)).findById(Mockito.any());
+            verify(menuGroupRepository, times(1)).findById(Mockito.any());
+
+
+            assertAll(
+                    () -> assertThat(menu.getId()).isNotNull(),
+                    () -> assertThat(menu.getMenuGroup().getId()).isEqualTo(menuRequest.getMenuGroupId()),
+                    () -> assertThat(menu.getName()).isEqualTo(menuRequest.getName()),
+                    () -> assertThat(menu.getMenuProducts()).hasSize(menuRequest.getMenuProducts().size()),
+                    () -> assertThat(menu.isDisplayed()).isEqualTo(menuRequest.isDisplayed()),
+                    () -> assertThat(menu.getPrice()).isEqualTo(menuRequest.getPrice())
+            );
+
+        }
+
+        @DisplayName(value = "메뉴의 가격은 0원 이상이어야 한다.")
+        @Test
+        void invalidMenuAmount() {
+            //상품명이나 가격이 없을때 에러처리
+            Menu menu = MenuTest.createMenu(후라이드치킨_MINUS_PRICE);
+            ThrowingCallable throwingCallable = () -> menuService.create(menu);
+            assertThatIllegalArgumentException().isThrownBy(throwingCallable);
+        }
+
+        @DisplayName(value = "메뉴의 주문 상품이 1개 이상 존재해야 합니다")
+        @Test
+        void invalidMenuProduct() {
+            MenuGroup menuGroup = createMenuGroup(한마리메뉴_MENU_GROUP_NAME, 후라이드치킨_MENU_GROUP_UUID);
+            menuGroupRepository.save(menuGroup);
+            Menu menu = MenuTest.createMenu(후라이드치킨_MENU_UUID, 후라이드치킨_MENU_NAME, 후라이드치킨_DEFAULT_PRICE, menuGroup, 후라이드치킨_MENU_GROUP_UUID);
+            ThrowingCallable throwingCallable = () -> menuService.create(menu);
+            assertThatIllegalArgumentException().isThrownBy(throwingCallable);
```

Comment:
(ThrowingCallable) 람다식을 꼭 변수로 추출할 필요는 없을 것 같아요. 👀 


또한 추가적으로 예외 검증은 `Assertions.assertThatThrownBy` 함수를 활용해볼 수 있을 것 같아요! 😃 

```suggestion
            assertThatThrownBy(() -> menuGroupService.create(menuGroup))
                .isExactlyInstanceOf(IllegalArgumentException.class);
```

Reply:
넵넵 일단 람다식 추출은 제거 처리 해보았습니다

예외검증의 경우, Illegal 예외가 확실하면 assertThatIllegalArgumentException를 그대로 사용해도 문제 없을 것 같아서 일단 그렇게 두었는데요,
assertThatThrownBy를 권장해주신 이유는 추후 예외가 변경될 수 있는 상황의 범용성 때문일까요?? 그게 맞다고 하시면 변경하는 게 좋겠다는 생각도 듭니다

Reply:
구체적 예외 케이스에 특화된 assertion 함수들은 importing을 여러번 해줘야하는 번거로움이 있어서, 지금같이 범용적으로 쓰는게 조금 더 좋다고생각했어요. 
하지만 이 부분은 사소한거라 엄청 권장한다거나 그런건 아니였어요. 😃
이런 것도 있다~ 정도로 봐주시면 좋을 것 같아요. 😉

Reply:
친절한 답변 감사드립니다 리뷰어님 (_ _)